earth_enterprise/src/server/wsgi/serve/push/search/core/search_push_manager.py

```diff
@@ -522,11 +530,13 @@ def __UpdatePoiStatus(self, file_paths, file_sizes, client_host_name):
           serve_utils.FileSizeMatched(server_file_path, file_sizes[i])):
 
         existing_files.append(file_paths[i])
+        logger.debug("%s already exists", server_file_path)
 
         file_paths.pop(i)
         file_sizes.pop(i)
       else:
         i += 1
+        logger.debug("%s does not exists", server_file_path)
```

Comment:
Minor: take the ending 's' off of 'exists'

Reply:
Good catch